### PR TITLE
EUI-4872 WA LD theme changes

### PR DESCRIPTION
**Pull request by johnbenjamin-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-4872

### Change description ###
LD Themes changes for WA

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/noc/noc.routes.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ export const ROUTES: Routes = [
       {
         path: '',
         component: null,
-        canActivate: [ HealthCheckGuard, NocGuard ],
+        canActivate: [ HealthCheckGuard, FeatureToggleGuard ],
```
Comment (udaydenduluri33) [inline src/noc/noc.routes.ts:16]: NocGuard removed?

Comment (johnbenjamin-hmcts) [reply src/noc/noc.routes.ts:16]: Yes @udaydenduluri33, @ronaldmansveld removed it when he was doing the development for refactoring header EUI-4575.

**Review by udaydenduluri33 — APPROVED** [review]

File: src/app/containers/app-header/app-header.component.ts [hunk 2]
```diff
@@ -2,14 +2,13 @@ import { Component, OnDestroy, OnInit } from '@angular/core';
 import { NavigationEnd, Router } from '@angular/router';
 import { FeatureToggleService } from '@hmcts/rpx-xui-common-lib';
 import { select, Store } from '@ngrx/store';
-import { combineLatest, Observable, of, Subscription } from 'rxjs';
-import { Theme, UserTypeRole } from '../../../app/models/theme.model';
-import { UserDetails } from '../../../app/models/user-details.model';
-
+import { Observable, of, Subscription } from 'rxjs';
+import { first } from 'rxjs/operators';
+import { ApplicationTheme, NavigationItem } from 'src/app/models/theming.model';
```
Comment (connorpgpmcelroy) [inline src/app/containers/app-header/app-header.component.ts:7]: src used here, need to do ../../../app/models/theming.model and same with below

File: src/app/containers/app-header-signed-out/app-header-signed-out.component.ts [hunk 3]
```diff
@@ -6,6 +6,7 @@ import {AppConstants} from '../../app.constants';
 import {AppTitleModel} from '../../models/app-title.model';
 import {NavItemsModel} from '../../models/nav-item.model';
 import {UserNavModel} from '../../models/user-nav.model';
+import { ApplicationTheme } from 'src/app/models/theming.model';
```
Comment (connorpgpmcelroy) [inline src/app/containers/app-header-signed-out/app-header-signed-out.component.ts:9]: src here as well

**Review by connorpgpmcelroy — CHANGES_REQUESTED** [review]
Need to remove references to src and use ../../ to find correct files

File: src/app/models/theming.model.ts [hunk 4]
```diff
@@ -0,0 +1,19 @@
+export interface ApplicationTheme {
+    appTitle: { name: string; url: string; };
+    backgroundColor: string;
+    logo: 'judicial' | 'myhmcts' | 'default' | 'none';
```
Comment (kuda1992) [inline src/app/models/theming.model.ts:4]: Can we extract this into an enum

**Review by kuda1992 — CHANGES_REQUESTED** [review]
One minor change but looks good to me

File: src/app/models/theming.model.ts (hunk 4 above)
Comment (johnbenjamin-hmcts) [reply src/app/models/theming.model.ts]: Thank you @kuda1992, I have now implemented the changes as you suggested.

File: src/app/containers/app-header/app-header.component.ts (hunk 2 above)
Comment (johnbenjamin-hmcts) [reply src/app/containers/app-header/app-header.component.ts]: Thank you @connorpgpmcelroy, I have now fixed this.

**Review by connorpgpmcelroy — APPROVED** [review]
Comments addressed, LGTM

**Review by kuda1992 — APPROVED** [review]

**Pull request merged**